xgcm/grid.py

```diff
@@ -1200,18 +1221,27 @@ def __init__(
             for key, value in metrics.items():
                 self.set_metrics(key, value)
 
-    def _parse_axes_kwargs(self, kwargs):
-        """Convvert kwarg input into dict for each available axis
+        # Finish setup
+
+    def _as_axis_kwarg_mapping(
+        self,
+        kwargs: Union[str, float, int, Dict[str, Union[str, float, int]]],
+        axes: Optional[list] = None,
+    ):
```

Comment:
You can specify a return type like this
```suggestion
    ) -> Dict[str, Union[str, float, int]]:
```

Reply:
Ah yes, totally forgot that. Thanks.